InternshipProject#2/Manager/IWatcherManager.cs

```diff
@@ -6,6 +6,6 @@ namespace InternshipProject_2.Manager
 {
     public interface IWatcherManager
     {
-        Task<WatchResponse> WatchTicket(WatchRequest request, int userId);
+        Task<WatchResponse> WatchTicket(WatchRequest request, int? userId);
```

Comment:
why nullable?

Reply:
Because GetClaimValue() from the TokenHelper can return a null, if the user is not logged in

Reply:
Wrong perspective...
You have to see this parameter in the WatchTicket method's context.
Can the WatchTicket method accept a null value there?
I don't think so. What do you do if the value is null?
It doesn't matter where you take the value from. If the GetClaimValue() returns a nullable value, it has nothing to do with this method. Before invoking the WatchTicket method you have to make sure the value you pass to it is not null, otherwise you won't call the method.